### PR TITLE
fix(table-sheet): 修复明细表 tooltip 展示了错误的汇总数据的问题

### PR DESCRIPTION
**Pull request by lijinke666**
### 👀 PR includes

🐛 Bugfix

- [x] Solve the issue and close #0

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| image | image |
| image |  image |
| image |  image | 

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/facet/table-facet.ts [hunk 1]
```diff
@@ -256,11 +256,11 @@ export class TableFacet extends FrozenFacet {
       }
 
       if (showSeriesNumber && col.field === SERIES_NUMBER_FIELD) {
-        data = rowIndex + 1;
+        data = `${rowIndex + 1}`;
```
Comment (lijinke666) [inline packages/s2-core/src/facet/table-facet.ts:259]: 和透视表统一, 序号是 string

File: packages/s2-core/src/data-set/table-data-set.ts [hunk 2]
```diff
@@ -156,17 +156,33 @@ export class TableDataSet extends BaseDataSet {
 
     const rowData = this.displayData[query.rowIndex];
 
-    if (!('col' in query) || !isObject(rowData)) {
```
Comment (lijinke666) [inline packages/s2-core/src/data-set/table-data-set.ts:159]: 统一使用 `field`, 已经有 `col` `key` `label` `value`, 太多奇怪的 key 了

**Review by wjgogogo — APPROVED** [review]

File: packages/s2-core/src/cell/base-cell.ts [hunk 3]
```diff
@@ -135,6 +136,11 @@ export abstract class BaseCell<T extends SimpleBBox> extends Group {
     return this.actualText;
   }
 
+  // TODO: 2.0 使用 G 内置的方法
+  public isTextOverflowing() {
+    return this.getActualText().includes(ELLIPSIS_SYMBOL);
```
Comment (YardWill) [inline packages/s2-core/src/cell/base-cell.ts:141]: this.getActualText()可能是undefined，这里includes会报错影响功能

**Pull request merged**